src/components/tasks/card/card.module.scss

```diff
@@ -35,7 +36,7 @@ $seeMoreTasksShadow: #595959cc;
     justify-content: space-between;
     border: 1px solid #d9d9d9;
     position: relative;
-    max-width: 720px;
+    width: 1050px;
```

Comment:
Width too wide can you decrease the width?

Reply:
How are we handling responsiveness here

Reply:
This issue was raised to fix the inconsistent width but okay I will handle the responsiveness as well.

Reply:
> Width too wide can you decrease the width?
I changed it on purpose to match it with the task-tab-container width.

Reply:
> How are we handling responsiveness here

This issue was raised to fix the inconsistent width but okay I will handle the responsiveness as well.

Reply:
> Width too wide can you decrease the width?

Design wise it matches the width of task-tabs-container and hence the change.